napari/_qt/_tests/test_plugin_widgets.py

```diff
@@ -97,18 +97,14 @@ def test_plugin_widgets_menus(test_plugin_widgets, make_napari_viewer):
             # this is the separator
             break
     actions = actions[cnt + 1 :]
-    assert len(actions) == 3
-    expected_text = ['TestP1', 'TestP2: Widg3', 'TestP3: magic']
-    assert [a.text() for a in actions] == expected_text
+    texts = [a.text() for a in actions]
+    for t in ['TestP1', 'TestP2: Widg3', 'TestP3: magic']:
+        assert t in texts
 
     # the first item of the plugins is a submenu (for "Test plugin1")
```

Comment:
Looks like the comment won't be quite right after the change.
```suggestion
    # Expect a submenu ("Test plugin1") with particular entries.
```

Reply:
Should this change be accepted?